src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```diff
@@ -87,6 +87,9 @@ main.opt.ignore-signing-information=\
 \                                        signed modular JARs are not copied to\n\
 \                                        the runtime image.
 
+main.opt.generate-cds-archive=\
```

Comment:
This should not be needed.  Leftover from an early version.

Reply:
Removed.